local-tests/test_flood.py

```diff
@@ -0,0 +1,63 @@
+#!/bin/env python
+import os
+import subprocess
+import sys
+from os.path import abspath, join
+from time import sleep, time
+
+from chainrunner import Chain, Seq, generate_keys
+
+# Path to working directory, where chainspec, logs and nodes' dbs are written:
+workdir = abspath(os.getenv('WORKDIR', '/tmp/workdir'))
+# Path to the pre-update aleph-node binary:
+bin = abspath(os.getenv('BINARY', join(workdir, 'aleph-node-old')))
```

Comment:
Technically we want to have the same behavior for this script in the case of `BINARY` or `FLOODER` envs missing, ie exiting with non-zero exit code? ie as `test_flood.sh` does.

Reply:
the file exist check will throw an error in such a case